content/en/docs/concepts/policy/pod-security-policy.md

```diff
@@ -464,12 +464,12 @@ allowed prefix, and a `readOnly` field indicating it must be mounted read-only.
 For example:
 
 ```yaml
-allowedHostPaths:
+  allowedHostPaths:
   # This allows "/foo", "/foo/", "/foo/bar" etc., but
   # disallows "/fool", "/etc/foo" etc.
   # "/foo/../" is never valid.
-  - pathPrefix: "/foo"
-    readOnly: true # only allow read-only mounts
+    - pathPrefix: "/foo"
```

Comment:
please indent the above comments as well, or else it is confusing which elements are targeted by the comments.

Reply:
@tengqm Ok, thanks for the tip. Can you check if now is ok :) ?